build/templates/session.py.mako

```diff
@@ -249,11 +249,14 @@ if grpc_supported:
         ${helper.get_function_docstring(ctor_for_docs, False, config, indent=8)}
         '''
 % if grpc_supported:
-        if _grpc_channel:
+        if _grpc_options:
             import ${module_name}._grpc_stub_interpreter as _grpc_stub_interpreter
-            interpreter = _grpc_stub_interpreter.GrpcStubInterpreter(_grpc_channel)
+            import ${module_name}.GrpcOptions as GrpcOptions
```

Comment:
Unused import

Reply:
Is it unnecessary because the stub interpreter already imports it? The options are used here in this file (session) for the auto-close behavior for example.

Reply:
You don't ever directly use anything from inside that module. If you never use `module_name.Member` in your code, you don't need to import `module_name` at all.

You accept a `_grpc_options` parameter constructed by the user, but when you access it you'll just use duck typing to grab the fields you need. So even in the final version of this code, I don't think you'll ever need to import the module except in the examples and in tests.